pkg/runtime/wfengine/activity.go

```diff
@@ -162,6 +163,7 @@ func (a *activityActor) executeActivity(ctx context.Context, actorID string, nam
 	//       introduce some kind of heartbeat protocol to help identify such cases.
 	callback := make(chan bool)
 	wi.Properties[CallbackChannelProperty] = callback
+	wfLogger.Debugf("activity actor '%s': scheduling activity %s for workflow with instanceId '%s'", actorID, name, wi.InstanceID)
```

Comment:
can you add the single quotes around the second %s, the one after scheduling activity, to be consistent with the logging throughout the rest of this PR?

Reply:
Done